string location_name
---
bool result